newrelic-integration-e2e/internal/settings.go

```diff
@@ -89,6 +90,18 @@ func SettingsWithAgentEnabled(agentEnabled bool) SettingOption {
 	}
 }
 
+func SettingsWithRegion(region string) SettingOption {
+	return func(o *settingOptions) {
+		// Using a Makefile as a wrapper makes region to be initialized to empty.
+		// So we make the default here.
+		if region != "" {
```

Comment:
I would instead, since the method is called SettingsWithRegion I would just leave here :
`o.region = region`
as Guillermo says but move the default to the getter function instead of the NewSettings as you can see in a comment bellow.

Reply:
Makes sense.